packages/video_player/video_player/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Flutter plugin for displaying inline video with other Flutter
   widgets on Android, iOS, and web.
 repository: https://github.com/flutter/packages/tree/main/packages/video_player/video_player
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+video_player%22
-version: 2.6.0
+version: 2.6.1
```

Comment:
This will also need to be 2.7.0

Reply:
Changed to 2.8.0 because current version is 2.7.1